.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
```

Comment:
I am wary of explicit version requirements for CI stuff because it ends up being something to chase.  Is there a way to indicate "follow stable" or something like that?

Reply:
There actually isn't a (easy) way to do this by design, https://pre-commit.com/#using-the-latest-version-for-a-repository. In general I think it is better to keep the versions of pinned for the sake of toolchain reproducibility. There is `pre-commit autoupdate` to update the `rev`s there to the latest versions, so really they are akin to entries in a lockfile.

Reply:
Bleh, ok.  I like the idea of having it track whatever that project is calling their stable release or release or whatever rather than experiencing bit rot, but maybe there's no way around that.